lms/djangoapps/courseware/features/problems.feature

```diff
@@ -216,3 +216,30 @@ Feature: LMS.Answer problems
         | multiple choice | incorrect          | correct          |
         | radio           | correct            | incorrect        |
         | radio           | incorrect          | correct          |
+
+    Scenario: I cannot see my score on a problem when correctness is never shown
```

Comment:
Lettuce tests are our legacy acceptance tests. We have been working for some time to convert them over to bok choy-- we definitely don't wish to add new lettuce tests.

In bok choy, problems are tested in test_problem_types.py (which tests individual problem types, using a mixin for testing common behavior) and test_lms_problems.py (which is more about general problem infrastructure, such as hints and notifications).